tests/test_tree_rings.py

```diff
@@ -0,0 +1,49 @@
+"""
+Unit tests for tree rings code.  Just tests that the tree ring data was found and could be read.
+"""
+import os
+import unittest
+import numpy as np
+import lsst.utils as lsstUtils
+from lsst.sims.GalSimInterface import make_galsim_detector
+from lsst.sims.GalSimInterface import LSSTCameraWrapper
+from lsst.sims.GalSimInterface import GalSimInterpreter
+import desc.imsim
+
+
+class TreeRingsTestCase(unittest.TestCase):
+    "TestCase class for the tree rings code."
+    def setUp(self):
+        self.sensors = ['R:2,2 S:1,1', 'R:3,4 S:2,2']
+        self.instcat_file = os.path.join(lsstUtils.getPackageDir('imsim'),
+                               'tests', 'tiny_instcat.txt')
+        self.rtest = 5280.0 # Just a value to test the radial function at
+        self.rvalues = [.0030205, -.0034135] # Expected results
+        self.centers = [(-3026.3, -3001.0), (3095.5, -2971.3)] # Input center values
+
+    def test_read_tree_rings(self):
+        "Check reading of tree_ring_parameters file"
```

Comment:
This should be enclosed in `"""` to be a docstring.

Reply:
Thanks for pointing this out! I did not know this...I've been making the same error elsewhere.

Reply:
Actually... I was wrong.  What you have is fine.   I have always been using `"""` and just assumed that was because I had to.  Sorry for the officiousness.

Reply:
I guess my misconception was based on a PEP8 convention

https://www.python.org/dev/peps/pep-0257/

Reply:
ok. [PEP 257](https://www.python.org/dev/peps/pep-0257/) does say to use `"""`, even for one-liners, so I plan to follow that convention.